src/core/webview/ClineProvider.ts

```diff
@@ -2422,7 +2416,7 @@ export class ClineProvider implements vscode.WebviewViewProvider {
 
 	// secrets
 
-	private async storeSecret(key: SecretKey, value?: string) {
+	public async storeSecret(key: SecretKey, value?: string) {
```

Comment:
We call this from the integration test, so it has to be public, but it seems like it shouldn't be. I'll try to clean this up in a follow-up.